verticapy/sql/functions/random.py

```diff
@@ -25,13 +25,39 @@ def random() -> StringSQL:
     -------
```

Comment:
**For everything below:**

Same as above:

change verticapy.stats by verticapy.sql.functions as vpf
bring the comment of the function usage out the code (create a new block)
change TableSample by vDataFrame (so you have to change your import as well)
Try to follow PEP
Use a new line after the last import